package.json

```diff
@@ -7,25 +7,24 @@
     "pnpm": "^9.1.0"
   },
   "devDependencies": {
+    "@biomejs/biome": "^1.9.4",
     "@changesets/cli": "^2.27.10",
+    "@ianvs/prettier-plugin-sort-imports": "^4.4.0",
+    "@nuxt/eslint-config": "^0.7.3",
     "@playwright/test": "^1.49.1",
-    "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "@types/node": "^20.17.10",
-    "@typescript-eslint/eslint-plugin": "^6.21.0",
-    "@typescript-eslint/parser": "^6.21.0",
     "@vitest/coverage-v8": "^1.6.0",
-    "@vue/eslint-config-typescript": "^12.0.0",
-    "eslint": "^8.57.0",
+    "@vue/eslint-config-typescript": "^14.1.3",
+    "biome": "link:@biomejs/cli-linux-x64/biome",
```

Comment:
https://github.com/biomejs/biome/issues/5109 Looks like this issue could be from a broken lockfile